recipes/kompose/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set organization = "kubernetes" %}
+{% set repository = "kompose" %}
+{% set version = "1.28.0" %}
+
+package:
+  name: {{ repository }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ organization }}/{{ repository }}.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0 
+  script:
```

Comment:
Please generate a list of third party licenses using go-licenses. See other go recipes as an example.